spec/models/ability_spec.rb

```diff
@@ -10,15 +10,15 @@
 
     context "as a guest" do
       let(:user){ nil }
-      it{ should     be_able_to(:read, circle) }
+      xit{ should     be_able_to(:read, circle) }
```

Comment:
Currently I am not sure what a 'guest' would signify. I think soon we will have new users that have not been approved by the circle administrator and those would have limited view right to the circle info. They should be able to see basic circle data but nothing more (i.e. sign up for tasks or anything like that).